client/src/components/Header.tsx

```diff
@@ -61,6 +61,7 @@ const UserMenu = styled.div`
   .bp3-button-text {
     overflow: hidden;
     text-overflow: ellipsis;
+    white-space: nowrap;
```

Comment:
this fixes an unrelated bug where long user emails with a hyphen in them were wrapping onto two lines